packages/suite/src/components/wallet/CoinjoinSummary/CoinJoinCustomStrategy.tsx

```diff
@@ -0,0 +1,211 @@
+    text-align: center;
+`;
+
+const RightMark = styled(Text)`
+    flex: 1 1 33.5%;
+    text-align: right;
+`;
+
+const MiningFee = styled.section`
+    flex-grow: 1;
+`;
+
+const sliderTrack = css`
+    background: linear-gradient(
+        270deg,
+        #bf6767 0%,
+        #c8b882 18.73%,
+        #c8b883 36.25%,
+        #95cda5 43.99%,
+        #2a9649 100%
+    );
+    cursor: pointer;
+    height: 3px;
+`;
+const sliderThumb = css`
+    appearance: none;
+    background: #ffffff;
+    border: ${({ theme }) => `1px solid ${theme.STROKE_GREY}`};
+    border-radius: 50%;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2);
+    cursor: grab;
+    height: 26px;
+    margin-top: -12px;
+    width: 26px;
+
+    :active {
+        box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+        cursor: grabbing;
+    }
+`;
+
+const Range = styled.input`
+    appearance: none;
+    margin: 30px 0 12px 0;
+    vertical-align: top; /* prevent extra bottom space in Firefox */
+    width: 100%;
+
+    ::-webkit-slider-runnable-track {
+        ${sliderTrack}
+    }
+    ::-webkit-slider-thumb {
+        ${sliderThumb}
+    }
+    ::-moz-range-track {
+        ${sliderTrack}
+    }
+    ::-moz-range-thumb {
+        ${sliderThumb}
+    }
+`;
+
+const DetailWrapper = styled.section`
+    background: ${({ theme }) => theme.BG_GREY};
+    border-radius: 4px;
+    flex-basis: 35%;
+    padding: 16px;
+`;
+
+const DetailHeading = styled.div`
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    margin-bottom: 16px;
+    text-transform: uppercase;
+`;
+
+const StyledSwitch = styled(Switch)`
+    margin-top: 2px;
+`;
+
+interface CoinjoinCustomStrategyProps {
+    reset: () => void;
+}
+
+export const CoinjoinCustomStrategy = ({ reset }: CoinjoinCustomStrategyProps) => {
+    const [maxFee, setMaxFee] = useState(3);
+    const [skipRounds, setSkipRounds] = useState(true);
+
+    const handleSliderChange: React.ChangeEventHandler<HTMLInputElement> = e =>
+        setMaxFee(Number(e.target.value));
+    const toggleSkipRounds = () => setSkipRounds(!skipRounds);
+
+    return (
+        <>
+            <TopRow>
+                <SetupHeading>
+                    <Translation id="TR_CUSTOM_SETUP" />
+                </SetupHeading>
+                <Button variant="tertiary" icon="REFRESH" onClick={reset}>
+                    <Translation id="TR_RESET_TO_DEFAULT" />
+                </Button>
+            </TopRow>
+            <MiddleRow>
+                <MiningFee>
+                    <Row>
+                        <Heading>
+                            <Translation id="TR_MAX_MINING_FEE" />
+                        </Heading>
+                        <Fee>{maxFee} sat/vB</Fee>
+                    </Row>
+                    <MiningFeeText>
+                        <Translation id="TR_MINING_FEE_NOTE" />
+                    </MiningFeeText>
+                    {/* TODO: separate Range component */}
```

Comment:
IMO it's better to at least abstract what you have now to something like `CoinJoinFeeRange` if you do not wanna create a global Range component :)

Reply:
I created a global `Range` component - 3770dcd2737b99b21f4c2d835c606a32271e4214